src/components/TheNavbar.vue

```diff
@@ -22,23 +22,61 @@
       </ul>
     </div>
   </nav>
-
 </template>
 
 <script>
 export default {
   name: 'TheNavbar',
+  
+  data() {
+    return {
+      pushRight: true,
+      screenWidth: 0,
+    }
+  },
+  created() {
+        window.addEventListener('resize', this.handleResize);
+        this.handleResize();
+  },
+  unmounted() {
+        window.removeEventListener('resize', this.handleResize);
+  },
+  methods: {
+    handleResize() {
+      this.screenWidth = window.innerWidth;
+    }    
+  },
+  watch: {
+    screenWidth(val){
+      if(val < 960){
+        this.pushRight = false;
+      }
+      else 
+        this.pushRight = true;
+    }
+  }
+  
```

Comment:
@phann-phearoen 

You could remove this part and other codes related to resizing with the use of Vuetify tags.